data/chain-store/sge-network-testnet.json

```
@@ -0,0 +1,57 @@
+{
+  "rpc": "https://rpc.staging.sixsigmasports.io",
+  "rest": "https://api.staging.sixsigmasports.io",
+  "chainId": "stage-sgenetwork",
+  "chainName": "stage-sgenetwork",
```

This field will be displayed to the end user, so we want something in the form of a title, for instance "Stage SGE Network"

```suggestion
  "chainName": "Stage SGE Network",
```